ios/RNMagicScript/components/Utils/Dialog/DialogView.swift

```diff
@@ -27,7 +27,8 @@ class DialogView: UIView {
     }
 
     static var height: CGFloat {
-        return DialogView.width * 0.75
+        let margin: CGFloat = 16.0
+        return max(UIScreen.height, UIScreen.width) - 2 * margin
```

Comment:
won't the height here be a lot bigger now?

Reply:
It will, but it does not matter. The height will be reduced by auto-layout. For some reason, it's easier for auto-layout to shrink the view rather than to expand it. I tested it on small and bigger screens (both portrait and landscape) and works fine.